src/operator/tensor/la_op.cc

```diff
@@ -32,6 +32,7 @@ DMLC_REGISTER_PARAMETER(LaMatrixMultParam);
 DMLC_REGISTER_PARAMETER(LaTriangMatrixMultParam);
 
 NNVM_REGISTER_OP(linalg_gemm)
+.add_alias("_linalg_gemm")
```

Comment:
Put the _linalg_* as name and linalg_* as alias. Later we will hide the aliases from documentation.

Reply:
Done. I did the same thing to random as well. Please let me know if it's not the right thing to do.
https://github.com/apache/incubator-mxnet/pull/7604/files#diff-9ceba26249f7709642ab151428dce746L48